src/PowerSimulations.jl

```diff
@@ -128,16 +129,16 @@ export set_service_model!
 export get_transmission_model
 ## Results interfaces
 export SimulationResultsExport
-export StageResultsExport
+export ProblemResultsExport
```

Comment:
same here

Reply:
The user can create this, so I think it should be exported.